fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -785,6 +786,10 @@ createImageClause
     : CREATE IMAGE
     ;
 
+cleanTabletSchedQClause
+    : CLEAN TABLET SCHEDULER QUEUE
```

Comment:
`BALANCE QUEUE`
TABLET SCHEDULER QUQUE can also refer to another schema change/ingestion

Reply:
balance is inaccurate, because we also have to schedule repair task, `TABLET SCHEDULER QUQUE`  should be enough, because we use the same prefix with confs in fe.conf